assets/src/logging.js

```diff
@@ -1,3 +1,4 @@
+'use strict';
```

Comment:
`'use strict';` is an environment directive that is parsed by the browser itself. It will enforce the code to be executed in a strict context, which means that the Javascript engine will be less forgiving.

See here for details: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode